src/tink/CoreApi.hx

```diff
@@ -13,7 +13,6 @@ typedef FutureTrigger<T> = tink.core.Future.FutureTrigger<T>;
 typedef PromiseTrigger<T> = tink.core.Promise.PromiseTrigger<T>;
 
 typedef Outcome<D, F> = tink.core.Outcome<D, F>;
-typedef OutcomeTools = tink.core.Outcome.OutcomeTools;

```

Comment:
Let's still expose it. There are some libraries using it. e.g. https://github.com/haxetink/tink_sql/blob/5b6c48a65ac4672abb969430ca08155008599012/src/tink/sql/drivers/sys/StdConnection.hx#L68

Reply:
Yeah...
https://github.com/haxetink/tink_macro/blob/8c50dcf04bbe4e693ecf3b3c48e9ad838f3f0aca/src/tink/MacroApi.hx#L22

Reply:
Done.